internal/dao/stubs/reservation_dao.go

```diff
@@ -2,11 +2,14 @@ package stubs
 
 import (
 	"context"
+	"errors"
 
 	"github.com/RHEnVision/provisioning-backend/internal/dao"
 	"github.com/RHEnVision/provisioning-backend/internal/models"
 )
 
+var NotImplementedErr = errors.New("stub not yet implemented")
+
```

Comment:
Nitpick: This could be in another `errors.go` too. A second file, it's a different package.